src/NUnitFramework/Directory.Packages.props

```diff
@@ -3,21 +3,18 @@
     <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
     <EnablePackageVersionOverride>false</EnablePackageVersionOverride>
   </PropertyGroup>
-
   <!-- Packages for used features -->
   <ItemGroup>
     <PackageVersion Include="System.Collections.Immutable" Version="6.0.0" />
     <PackageVersion Include="System.ValueTuple" version="4.5.0" />
   </ItemGroup>
-
   <!-- General Packages -->
   <ItemGroup>
-    <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="1.1.1" />
-    <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.7.2" />
+    <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="8.0.0" />
```

Comment:
This seems like a major bump, but I guess there are no breaking changes?

Reply:
Didn't look like no. And this is just for development ("There are no supported framework assets in this package.") , and we build using the 8 version.